views/NewsScreen.tsx

```diff
@@ -287,102 +270,137 @@ function NewsScreen({ navigation }: {
           layout={Layout.duration(250).easing(Easing.out(Easing.bezierFn(1, 0, 0.5, 1)))}
         >
           {!isLoading && news.length !== 0 && (
-            news.map((item, index) => (
-              !unreadOnly || !item.read || (unreadOnly && news.filter((item) => !item.read).length === 0) ? (
-                <Reanimated.View
+            news.map((item, index) => {
+              if (!swipeableRefs.current[index]) {
+                swipeableRefs.current[index] = React.createRef();
+              }
+
+              const swipeableRef = swipeableRefs.current[index];
+
+              return (
+                <Swipeable
                   key={item.author + item.date + item.title}
-                  style={[
-                    {
-                      overflow: 'hidden',
-                      borderRadius: 10,
-                      borderCurve: 'continuous',
-                      marginBottom: 8,
-                    },
-                  ]}
-                  layout={Layout.duration(250).easing(Easing.out(Easing.bezierFn(1, 0, 0.5, 1)))}
-                  entering={ZoomIn.duration(250).easing(Easing.out(Easing.bezierFn(0.5, 0, 1, 0)))}
-                  exiting={FadeOut.duration(200)}
-                >
-                  <NativeItem
-                    onPress={() => {
-                      navigation.navigate('NewsDetails', { news: item });
-                    }}
-                  >
-                    <View style={[{ gap: 4, marginLeft: 14 }]}>
-                      <View
+                  renderLeftActions={() => (
+                    <TouchableOpacity
+                      style={{
+                        backgroundColor: item.read ? 'white' : 'green',
+                        borderRadius: 10,
+                        height: item.height <= 101.5 ? '92%' : '94%', // C'est moche mais j'ai pas réussi à faire autrement
```

Comment:
Ça sert à quoi ? Je ne vois pas la différence sur mes actualités.

Reply:
Tu peux aussi modifier en:
```suggestion
                        height: item.height <= 101.5 ? '92%' : '94%', // C'est moche mais j'ai pas réussi à faire autrement
                        width: '25%'

```
Pour moi la taille était bonne, je te laisse tester de ton côté.